### PR TITLE
DOC: Updating str_repeat docstring

### PR DESCRIPTION
**Pull request by JesperDramsch**

### CODE REVIEW

**Comment by Bilingyun** [comment]
## hahahah

**Comment by gfyoung** [comment]
https://travis-ci.org/pandas-dev/pandas/jobs/423383576#L3419

@JesperDramsch : Looks like you have a linting failure here.

**Comment by gfyoung** [comment]
@JesperDramsch : I like the change!  FYI, Anaconda is having some serious servicing issues, so our CI is not going to work at all for the time being.

**Comment by JesperDramsch** [comment]
Alright then I'll chill out and enjoy the view.

File: pandas/core/strings.py [hunk 1]
```diff
@@ -675,20 +675,42 @@ def str_replace(arr, pat, repl, n=-1, case=None, flags=0, regex=True):
...
+        Series or Index of repeated string objects specified by
+        input parameter repeats.
+
+    Examples
+    --------
+    >>> s = pd.Series(['a', 'b', 'c'])
+    >>> s
+    0    a
+    1    b
+    2    c
```
Comment (jeffyancey) [inline pandas/core/strings.py]: I don't know that it is necessary, be I do see that elsewhere in our examples the `dtype: object` is included beneath the series printout.

File: pandas/core/strings.py [hunk 2]
```diff
@@ -675,20 +675,42 @@ def str_replace(arr, pat, repl, n=-1, case=None, flags=0, regex=True):
...
     """
-    Duplicate each string in the Series/Index by indicated number
-    of times.
+    Duplicate each string in the Series or Index.
 
     Parameters
     ----------
-    repeats : int or array
-        Same value for all (int) or different value per (array)
+    repeats : int or sequence of int
```
Comment (jeffyancey) [inline pandas/core/strings.py]: `sequence of int` might be fine, and maybe 'sequence' is a term that has a clear definition in the pandas community, but I often see `array_like` used in the numpy documentation. Not really recommending anything, just curious is `sequence` is used similarly here, or you were just using to to indicate a list/array/tuple/etc of integers.

**Review by jeffyancey — COMMENTED** [review]
Basically looks good to me, just a couple questions and observations.

**Comment by jreback** [comment]
@datapythonista

**Comment by datapythonista** [comment]
Finally merging this. We can add improvements from #20634 later on in a new PR.

**Comment by datapythonista** [comment]
Thanks @JesperDramsch

**Pull request merged**